ShenyuClient.sln

```diff
@@ -0,0 +1,86 @@
+﻿

```

Comment:
this can not add？

Reply:
I think it's better to be added. `*.sln` file is used to organize projects in .NET. Like the root `pom.xml` file. There're some projects for references.
- https://github.com/apache/pulsar-dotpulsar/blob/master/DotPulsar.sln
- https://github.com/open-telemetry/opentelemetry-dotnet/blob/9d843e6ce3/OpenTelemetry.sln